docs/extensions/webhook-remediator.md

```diff
@@ -0,0 +1,44 @@
+# Webhook remediator
+
+## Explanation
+As a Gardener deployment grows to host hundreds (if not thousands) of clusters, it becomes hard for the operators to effitially answer all support requests from cluster's users. One issue that has been recuring is related to validating/mutating webhooks : Since kubernetes provides webhook resource to validate / modify resources created by cluster's users, it is possible that a wrongly configured webhook interferes with Gardener's ability modify the cluster as required.
```

Comment:
The better place to add this doc is in `usage`.
WDYT? @rfranzke

Reply:
There is already some mentioning about the remediator in https://github.com/gardener/gardener/blob/master/docs/usage/shoot_status.md#constraints, perhaps we should enhance this section?

Reply:
Updated the existing doc as recommended!